src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -217,14 +218,24 @@ public static void UpdateReturnType(this AppCompatEditText editText, IEntry entr
 			editText.ImeOptions = entry.ReturnType.ToNative();
 		}
 
-		[PortHandler]
+		public static void UpdateAutoSize(this AppCompatEditText editText, IEditor editor)
+		{
+			if (editor.AutoSize == EditorAutoSizeOption.Disabled)
+			{
+				editText.SetAutoSizeTextTypeWithDefaults(AutoSizeTextType.None);
+			}
+			else
+			{
+				editText.SetAutoSizeTextTypeUniformWithConfiguration(editText.AutoSizeMinTextSize, editText.AutoSizeMaxTextSize, editText.AutoSizeStepGranularity, (int)ComplexUnitType.Sp);
```

Comment:
I don't think this is the correct API to use for this one.  Doesn't this API cause the font to autosize? We don't want the font to change size based on the size of the editor.  In the original implementation Android was able to just update its size by re-measuring so we might need to change the invalidation code.